### PR TITLE
Gidk 23ghp7 daybook test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 1]
```diff
@@ -246,15 +339,46 @@ <h4>3.1 Details of Outward Supplies and inward supplies liable to reverse charge
                             </tr>
                             <tr>
                                 <td class="bold">Total Value</td>
-                                <td class="text-right"><i class="icon-rupees font-11"></i>{{(this.gstr3BData?.sumTaxVal ? this.gstr3BData?.sumTaxVal : 0 )| giddhCurrency}}
+                                <td class="text-right"><i class="icon-rupees font-11"></i>
+                                    <!-- {{(this.gstr3BData?.sumTaxVal ? this.gstr3BData?.sumTaxVal : 0 )| giddhCurrency}} -->
+                                    <span class="d-inline-flex">
+                                        <amount-field
+                                            [amount]="(this.gstr3BData?.sumTaxVal ? this.gstr3BData?.sumTaxVal : 0)"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:346]:
```suggestion
                                            [amount]="(gstr3BData?.sumTaxVal ? gstr3BData?.sumTaxVal : 0)"
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 2]
```diff
@@ -246,15 +339,46 @@ <h4>3.1 Details of Outward Supplies and inward supplies liable to reverse charge
...
+                                            [currencySymbol]="false" [currencyCode]="false">
+                                        </amount-field>
+                                    </span>
                                 </td>
-                                <td class="text-right"><i class="icon-rupees font-11"></i>{{(this.gstr3BData?.sumIamtVal ? this.gstr3BData?.sumIamtVal : 0 )| giddhCurrency}}
+                                <td class="text-right"><i class="icon-rupees font-11"></i>
+                                    <!-- {{(this.gstr3BData?.sumIamtVal ? this.gstr3BData?.sumIamtVal : 0 )| giddhCurrency}} -->
+                                    <span class="d-inline-flex">
+                                        <amount-field [amount]="(this.gstr3BData?.sumIamtVal ? this.gstr3BData?.sumIamtVal : 0 )" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:354]:
```suggestion
                                        <amount-field [amount]="(gstr3BData?.sumIamtVal ? gstr3BData?.sumIamtVal : 0 )" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 3]
```diff
@@ -246,15 +339,46 @@ <h4>3.1 Details of Outward Supplies and inward supplies liable to reverse charge
...
+                                            [currencyCode]="false">
+                                        </amount-field>
+                                    </span>
                                 </td>
-                                <td class="text-right"><i class="icon-rupees font-11"></i>{{(this.gstr3BData?.sumCamtval ? this.gstr3BData?.sumCamtval : 0 )| giddhCurrency}}
+                                <td class="text-right"><i class="icon-rupees font-11"></i>
+                                    <!-- {{(this.gstr3BData?.sumCamtval ? this.gstr3BData?.sumCamtval : 0 )| giddhCurrency}} -->
+                                    <span class="d-inline-flex">
+                                        <amount-field [amount]="(this.gstr3BData?.sumCamtval ? this.gstr3BData?.sumCamtval : 0 )" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:362]:
```suggestion
                                        <amount-field [amount]="(gstr3BData?.sumCamtval ? gstr3BData?.sumCamtval : 0 )" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 4]
```diff
@@ -246,15 +339,46 @@ <h4>3.1 Details of Outward Supplies and inward supplies liable to reverse charge
...
+                                            [currencyCode]="false">
+                                        </amount-field>
+                                    </span>
                                 </td>
-                                <td class="text-right"><i class="icon-rupees font-11"></i>{{(this.gstr3BData?.sumSamtval ? this.gstr3BData?.sumSamtval : 0 )| giddhCurrency}}
+                                <td class="text-right"><i class="icon-rupees font-11"></i>
+                                    <!-- {{(this.gstr3BData?.sumSamtval ? this.gstr3BData?.sumSamtval : 0 )| giddhCurrency}} -->
+                                    <span class="d-inline-flex">
+                                        <amount-field [amount]="(this.gstr3BData?.sumSamtval ? this.gstr3BData?.sumSamtval : 0 )" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:370]:
```suggestion
                                        <amount-field [amount]="(gstr3BData?.sumSamtval ? gstr3BData?.sumSamtval : 0 )" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 5]
```diff
@@ -246,15 +339,46 @@ <h4>3.1 Details of Outward Supplies and inward supplies liable to reverse charge
...
+                                            [currencyCode]="false">
+                                        </amount-field>
+                                    </span>
                                 </td>
-                                <td class="text-right"><i class="icon-rupees font-11"></i>{{(this.gstr3BData?.sumCsamtval ? this.gstr3BData?.sumCsamtval : 0 )| giddhCurrency}}
+                                <td class="text-right"><i class="icon-rupees font-11"></i>
+                                    <!-- {{(this.gstr3BData?.sumCsamtval ? this.gstr3BData?.sumCsamtval : 0 )| giddhCurrency}} -->
+                                    <span class="d-inline-flex">
+                                        <amount-field [amount]="(this.gstr3BData?.sumCsamtval ? this.gstr3BData?.sumCsamtval : 0 )" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:378]:
```suggestion
                                        <amount-field [amount]="(gstr3BData?.sumCsamtval ? gstr3BData?.sumCsamtval : 0 )" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 6]
```diff
@@ -393,19 +644,67 @@ <h4>4. Eligible ITC</h4>
...
+                                                <!-- {{item?.camt | giddhCurrency}} -->
+                                                <span class="d-inline-flex">
+                                                    <amount-field [amount]="item?.camt" [currencySymbol]="false" [currencyCode]="false">
+                                                    </amount-field>
+                                                </span>
+                                            </td>
+                                            <td class="text-right "><i class="icon-rupees font-11"></i>
+                                                <!-- {{item?.samt | giddhCurrency}} -->
+                                                <span class="d-inline-flex">
+                                                    <amount-field [amount]="item?.camt" [currencySymbol]="false" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:697]:
```suggestion
                                                    <amount-field [amount]="item?.samt" [currencySymbol]="false" [currencyCode]="false">
```

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 7]
```diff
@@ -476,19 +844,38 @@ <h4>5. Values of exempt, nil-rated and non-GST inward supplies </h4>
...
+                                                <span class="d-inline-flex">
+                                                    <amount-field [amount]="item?.inter" [currencySymbol]="false" [currencyCode]="false">
+                                                    </amount-field>
+                                                </span>
                                             </td>
-                                            <td class="text-right btn-link" data-ember-action="" data-ember-action-1233="1233"><i class="icon-rupees font-11"></i>{{item?.intra |giddhCurrency}}
+                                            <td class="text-right btn-link" data-ember-action="" data-ember-action-1233="1233"><i class="icon-rupees font-11"></i>
+                                                <!-- {{item?.intra |giddhCurrency}} -->
+                                                <span class="d-inline-flex">
+                                                    <amount-field [amount]="item?.intra " [currencySymbol]="false" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:857]:
```suggestion
                                                    <amount-field [amount]="item?.intra" [currencySymbol]="false" [currencyCode]="false">
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 8]
```diff
@@ -97,8 +97,22 @@
                             <td [attr.data-title]="commonLocaleData?.app_particular">{{ entry.particular.name }}</td>
                             <td [attr.data-title]="localeData?.voucher_name">{{ entry.voucherName }}</td>
                             <td class="text-right" [attr.data-title]="localeData?.voucher_no">{{ entry.voucherNo }}</td>
-                            <td class="text-right" [attr.data-title]="localeData?.debit_amount">{{ entry.debitAmount || 0 |giddhCurrency }}</td>
-                            <td class="text-right" [attr.data-title]="localeData?.credit_amount">{{ entry.creditAmount || 0 |giddhCurrency}}</td>
+                            <td class="text-right" [attr.data-title]="localeData?.debit_amount">
+                                <!-- {{ entry.debitAmount || 0 |giddhCurrency }} -->
+                                <span class="d-flex justify-content-end">
+                                    <amount-field [amount]="entry.debitAmount"
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.html:103]: Done

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 9]
```diff
@@ -97,8 +97,22 @@
...
+                                        [currencySymbol]="false"
+                                        [currencyCode]="false">
+                                    </amount-field>
+                                </span>
+                            </td>
+                            <td class="text-right" [attr.data-title]="localeData?.credit_amount">
+                                <!-- {{ entry.creditAmount || 0 |giddhCurrency}} -->
+                                <span class="d-flex justify-content-end">
+                                    <amount-field [amount]="entry.creditAmount" [currencySymbol]="false" [currencyCode]="false">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.html:112]: Done

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 10]
```diff
@@ -108,11 +122,23 @@
                                 <td class="hide-td"></td>
                                 <td class="hide-td"></td>
                                 <td class="text-right" [attr.data-title]="localeData?.debit_amount">
-                                    <span *ngIf="details.type === 'DEBIT'">{{ details.amount || 0 |giddhCurrency}}</span>
+                                    <span *ngIf="details.type === 'DEBIT'">
+                                        <!-- {{ details.amount || 0 |giddhCurrency}} -->
+                                        <span class="d-flex justify-content-end">
+                                            <amount-field [amount]="details.amount" [currencySymbol]="false" [currencyCode]="false">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.html:128]: Done

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 11]
```diff
@@ -108,11 +122,23 @@
...
+                                        </span>
+                                    </span>
                                     <span *ngIf="details.type !== 'DEBIT'">-</span>
                                 </td>
                                 <td class="text-right" [attr.data-title]="localeData?.credit_amount">
-                                    <span *ngIf="details.type === 'CREDIT'">{{ details.amount|| 0  |giddhCurrency}}</span>
+                                    <span *ngIf="details.type === 'CREDIT'">
+                                        <!-- {{ details.amount|| 0  |giddhCurrency}} -->
+                                        <span class="d-flex justify-content-end">
+                                            <amount-field [amount]="details.amount" [currencySymbol]="false" [currencyCode]="false">
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.html:138]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:346]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:354]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:362]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:370]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:378]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:697]: Done

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:857]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**